src/components/home/VolunteerBrief.jsx

```diff
@@ -4,110 +4,105 @@ import Button from '@mui/material/Button';
 import Card from '@mui/material/Card';
 import Grid from '@mui/material/Grid';
 import Typography from '@mui/material/Typography';
-// Other Library Imports
-import {
-  FaChartPie,
-  FaDatabase,
-  FaHandHoldingHeart,
-  FaHandshakeSimple,
-  FaLaptopCode,
-  FaPenRuler,
-  FaPeopleLine,
-  FaUser
-} from 'react-icons/fa6';
+// Material UI Icons Imports
+import DesignServicesOutlinedIcon from '@mui/icons-material/DesignServicesOutlined';
+import GroupsOutlinedIcon from '@mui/icons-material/GroupsOutlined';
+import HandshakeOutlinedIcon from '@mui/icons-material/HandshakeOutlined';
+import PersonOutlinedIcon from '@mui/icons-material/PersonOutlined';
+import QueryStatsOutlinedIcon from '@mui/icons-material/QueryStatsOutlined';
+import StorageRoundedIcon from '@mui/icons-material/StorageRounded';
+import TerminalOutlinedIcon from '@mui/icons-material/TerminalOutlined';
+import VolunteerActivismOutlinedIcon from '@mui/icons-material/VolunteerActivismOutlined';
 
 const volunteerGrid = [
   {
-    icon: <FaLaptopCode size={45} />,
+    icon: <TerminalOutlinedIcon fontSize="large" />,
     label: 'Developers',
     alt: 'Laptop with code'
   },
   {
-    icon: <FaDatabase size={45} />,
+    icon: <StorageRoundedIcon fontSize="large" />,
     label: 'Data Scientists',
     alt: 'Database'
   },
   {
-    icon: <FaPenRuler size={45} />,
+    icon: <DesignServicesOutlinedIcon fontSize="large" />,
     label: 'Designers',
     alt: 'Pen and ruler'
   },
   {
-    icon: <FaPeopleLine size={45} />,
+    icon: <GroupsOutlinedIcon fontSize="large" />,
     label: 'Project Managers',
     alt: 'People in a line'
   },
   {
-    icon: <FaChartPie size={45} />,
+    icon: <QueryStatsOutlinedIcon fontSize="large" />,
     label: 'Product Managers',
     alt: 'Pie chart'
   },
   {
-    icon: <FaHandshakeSimple size={45} />,
+    icon: <HandshakeOutlinedIcon fontSize="large" />,
     label: 'Social Workers',
     alt: 'Handshake'
   },
   {
-    icon: <FaHandHoldingHeart size={45} />,
+    icon: <VolunteerActivismOutlinedIcon fontSize="large" />,
     label: 'Community Organizers',
     alt: 'Hand holding heart'
   },
   {
-    icon: <FaUser size={45} />,
+    icon: <PersonOutlinedIcon fontSize="large" />,
     label: 'Anyone!',
     alt: 'User avatar'
   }
 ];
 
+const cardStyle = {
+  p: 4,
+  backgroundColor: 'rgba(217, 217, 217, 0.4)',
+  borderRadius: '30px'
+};
+
 const VolunteerBrief = () => {
   return (
     <Box
```

Comment:
Change line 68 `<Box>` to a `<Stack>` and it'll look like what you want:
<img width="1440" alt="Screenshot 2023-11-15 at 7 50 30 AM" src="https://github.com/codeforpdx/codepdx_website/assets/71395931/aeca1f72-54f1-4b68-bf1b-fe732731ba8c">

instead of this:
<img width="1440" alt="Screenshot 2023-11-15 at 7 49 45 AM" src="https://github.com/codeforpdx/codepdx_website/assets/71395931/0ff136b1-3a48-4821-8b0b-1fbb1142de3e">

Reply:
Oh weird. Could have sworn it wasn't doing that before. Fixed now.